react-basic-practice/src/components/Table/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Th,
+  Tr,
+  Thead,
+  Table as ChakraTable,
+} from "@chakra-ui/react";
+import SortingIcon from "../Icons/sorting";
```

Comment:
Pls create `index.ts` file for this folder

Reply:
i done